Tidy up entry

ilectra opened the pull request:
Removed un-needed, duplicated entry functions and scripts. In the process, some tidy up of other open issues:
Fixes #33 
Fixes #32 
Fixes #63  (Changes ported over from #65 )

jgomezdans commented on scripts/run_kaska:
```diff
@@ -1,33 +1,69 @@
+              'info':logging.INFO,
+              'warning':logging.WARNING,
+              'error':logging.ERROR }
+LOG = logging.getLogger("KaSKA")
```
you get use `get_logger` for this

ilectra replied:
Yes, much better this way, thanks!

jgomezdans commented on scripts/run_kaska:
```diff
@@ -1,33 +1,69 @@
+approx_inverter = get_inverter("prosail_5paras", "Sentinel2")
+dask_client = None
+
+kaska_runner(args.start_date, args.end_date, args.temporal_grid_space, args.state_mask,
```
On mobile, but I assume that you can just import kaska and call kaska_runner directly right?

ilectra replied:
Yes, you can. If anything this script is an example of exactly how to do that in your own script.
Having the CLI doesn't prevent anyone from using kaska as a library, as intended.

jgomezdans commented on scripts/run_kaska:
```diff
@@ -1,33 +1,69 @@
 #parent_folder = "/home/ucfajlg/Data/python/KaFKA_Validation/LMU/s2_obs/"
 #state_mask = "/home/ucfajlg/Data/python/KaFKA_Validation/LMU/carto/ESU.tif"
 
-run_process(args.start_date, args.end_date, args.temporal_grid_space,
```
I think you still want to have a `run_process` function exposed to the user (e.g. available via `import kaska`)

ilectra replied:
ok, you're right, I added it back, and will document it as well. This function should now be called by _ALL_ entry scripts, no more `__main__`'s.

pep8speaks commented:
Hello @ilectra! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`kaska/entry.py`](https://github.com/UCL/kaska/blob/f98960093d66f72576e7b22286499f1bce64cde9/kaska/entry.py):

> [Line 9:46](https://github.com/UCL/kaska/blob/f98960093d66f72576e7b22286499f1bce64cde9/kaska/entry.py#L9): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 31:17](https://github.com/UCL/kaska/blob/f98960093d66f72576e7b22286499f1bce64cde9/kaska/entry.py#L31): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent

ilectra commented:
@jgomezdans , I think I addressed your comments, please approve and merge if you're happy! Remember I implemented some of your changes from #65 as well.
